scripts/ensure-rustup.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e -x
```

Comment:
nit: I prefer the verbose form of these options:

```shell
#!/usr/bin/env bash

set -o errexit
set -o nounset
```

as it's easier to read later.